starter-code/iteration2/iteration2/src/app/components/main/main.component.html

```diff
@@ -0,0 +1,6 @@
+<div>
+  <h1> {{ title }} </h1>
+  <img src="{{"image"}}" alt="">
```

Comment:
`"{{"image"}}"` CARE!
Here you have a string with the value `image`. What i'm guessing that you wanted to have is `{{ image }}` without the quotes.